docs/cookbook/full-wallet.md

```diff
@@ -0,0 +1,43 @@
+# Full Wallet Example
+
+This page will do a walk through of 3 examples to illustrate core wallet functionality including:
```

Comment:
I would remove the reference to the examples. The fact that this page is cut into 3 examples when you look at the code is an implementation detail (although we could have a mention of it somewhere).

Overall I'd rather this page be conceptually "1 full continuous example", even if the code snippets it pulls from are in different places.

Reply:
This line could simply read:

_This page will do a walk through core wallet functionalities including:_

Reply:
I don't think I agree with that. If I'm reading docs and there are runnable examples I want to know that. we could reword it to make it clear you don't need to do that... but it's unclear to me the direction we should go here. we kinda have two options as I see it: don't link to example code and print it all in the docs, or pull only snippets into docs and link to code for dependencies, imports, etc... right now it's at a bit of a middle ground because I feel we haven't really made that decision yet. the quickstart page just does both, but the plan was to have that one be unique, so ya I would expect us to make that choice on the full wallet